src/sidebars/search/search-controller.js

```diff
@@ -21,7 +21,12 @@
         }
 
         select(stock) {
-            this.selectionService.select(stock);
+            if (!this.store) {
+                this.store = window.storeService.open(window.name);
+            }
+            if (!this.store.isCompact() && stock.favourite) {
+                this.selectionService.select(stock);
+            }
```

Comment:
The way Colin put it was that you shouldn't be able to select search results, which to me would mean you can't select things that aren't in your favourites, because it seems strange to be able to select stuff straight from search. If you allow selection from search you can easily get into a state where the full view is showing the graph for something not in your favourites. Do we want that to be possible?